runtime/jvmti/j9jvmti.tdf

```diff
@@ -461,6 +461,7 @@ TraceAssert=Assert_JVMTI_false noEnv Overhead=1 Level=1 Assert="!(P1)"
 TraceAssert=Assert_JVMTI_notNull noEnv Overhead=1 Level=1 Assert="(P1) != NULL"
 TraceAssert=Assert_JVMTI_mustHaveVMAccess noEnv overhead=1 Level=1 Assert="(P1)->publicFlags & J9_PUBLIC_FLAGS_VM_ACCESS"
 TraceAssert=Assert_JVMTI_mustNotHaveVMAccess noEnv overhead=1 Level=1 Assert="0 == ((P1)->publicFlags & J9_PUBLIC_FLAGS_VM_ACCESS)"
+TraceAssert=Assert_JVMTI_unreachable noEnv Overhead=1 Level=1 Assert="0"
```

Comment:
Tracepoints must be added at the end for backward-compatibility. See https://github.com/eclipse-openj9/openj9/blob/master/doc/diagnostics/AddingTracepoints.md.

Reply:
`Assert_JVMTI_true(0)` can be used instead of adding a new tracepoint.

Reply:
Yes, but then you couldn't distinguish this new situation.